### ISSUE TITLE
Bug in number

### ISSUE THREAD

**Issue by higimo** [Dok11/bitrix_module_als.typograf#2]
http://prntscr.com/f43rh8

Light typograf must ignore the html-code

Example for test:

```html
<div class="margin: 0 auto;">
  0 rub.
</div>
```

**Comment by Dok11** [comment]
![image](https://cloud.githubusercontent.com/assets/2697890/25715672/0056585a-3105-11e7-9102-2c80801519da.png)

P.S. Проще в доку написать что бы light использовался только для не-html :)
Вообще он задумывался для строк, которые в админке input type="text", что бы визивиг не городить.

**Comment by higimo** [comment]
А я думал, что ему можно всё скормить. Видимо, придётся хитрее типографить=(

**Comment by Dok11** [comment]
Да, с лайтовым так. В данный модуль я его воткнул прям вообще на крайний случай, если ни artlebedev типограф, ни с typograf.ru недоступны.
Изначально это был просто один маленький класс в als.helper прошлого проекта.

**Comment by Dok11** [comment]
Если нужно html типографить, то вот норм вариант: https://packagist.org/packages/kix/mdash
